apps/examples/testcase/le_tc/kernel/Kconfig

```diff
@@ -200,5 +208,65 @@ config TC_KERNEL_TASH_STACKMONITOR
 	bool "StackMonitor"
 	default n
 
+config ITC_KERNEL_ENVIRON
+	bool "ITC Environ"
+	default n
+if ITC_KERNEL_ENVIRON
+config ITC_LOOP_SIZE
+	int "Iterator Count"
+	default 100
+endif
+
+config ITC_KERNEL_LIBC_PTHREAD
+	bool "ITC libc Pthread"
+	default n
+if ITC_KERNEL_LIBC_PTHREAD
+config ITC_LOOP_SIZE
+	int "Iterator Count"
+	default 100
+endif
+
+config ITC_KERNEL_LIBC_SEMAPHORE
+	bool "ITC Libc Semaphore"
+	default n
+
+config ITC_KERNEL_SEMAPHORE
+	bool "ITC Semaphore"
+	default n
+	select DEBUG
+	select PRIORITY_INHERITANCE
+if ITC_KERNEL_SEMAPHORE
+config ITC_LOOP_SIZE
+	int "Iterator Count"
+	default 100
+endif
+
+config ITC_KERNEL_SCHED
+	bool "ITC Sched"
+	default n
+if ITC_KERNEL_SCHED
+config ITC_LOOP_SIZE
+	int "Iterator Count"
+	default 100
+endif
+
+config ITC_KERNEL_TIMER
+	bool "ITC Timer"
+	default n
+
+config ITC_KERNEL_LIBC_SPAWN
+	bool "ITC Libc Spawn"
+	default n
+
+config ITC_KERNEL_PTHREAD
+	bool "ITC Pthread"
+	default n
+	select MUTEX_TYPES
+if ITC_KERNEL_PTHREAD
+config ITC_LOOP_SIZE
+	int "Iterator Count"
+	default 100
+endif
```

Comment:
why config `ITC_LOOP_SIZE ` duplicated in above itc configs?

Reply:
same as lokesh.

Reply:
Fixed review comment